tests/system_tests_edge_router.py

```diff
@@ -2540,6 +2540,7 @@ def __init__(self, receiver1_host, receiver2_host, receiver3_host,
 
     def on_released(self, event):
         self.n_released += 1
+        self.send_test_message()
```

Comment:
There were two definitions of `on_released` in the same class. This is the case with many of the mypy warnings in this PR. Either duplicate method in class or duplicate class in module, with one instance imported `from proton import Timeout` and the other one defined there directly `class Timeout:`